src/lang/english.txt

```diff
@@ -1642,6 +1642,8 @@ STR_CONFIG_SETTING_ZOOM_MIN                                     :Maximum zoom in
 STR_CONFIG_SETTING_ZOOM_MIN_HELPTEXT                            :The maximum zoom-in level for viewports. Note that enabling higher zoom-in levels increases memory requirements
 STR_CONFIG_SETTING_ZOOM_MAX                                     :Maximum zoom out level: {STRING2}
 STR_CONFIG_SETTING_ZOOM_MAX_HELPTEXT                            :The maximum zoom-out level for viewports. Higher zoom-out levels might cause lag when used
+STR_CONFIG_SETTING_SPRITE_ZOOM_MIN                              :Maximum zoom level for sprites: {STRING2}
+STR_CONFIG_SETTING_SPRITE_ZOOM_MIN_HELPTEXT                     :The maximum zoom level at which sprites will be drawn (for custom graphics sets which provide additional zoom levels). Note that enabling higher zoom-in levels increases memory requirements.
```

Comment:
How does this affect the memory consumption?

Reply:
From my understanding, there would be a small additional memory usage from loading a 2x/4x sprite into the cache - but maybe this would just cause earlier cache eviction rather than additional memory usage. To be tested.

Reply:
In a test game there was a bump in memory usage of about 2MB the first time I changed from 1x to 2x with extra zoom sets enabled, although it did not reduce upon subsequently reducing it back to 1x.

Regardless, I prefer the messages suggested by neilsmh.